apps/nextjs/src/app/aila/help/index.tsx

```diff
@@ -204,8 +207,9 @@ const Help = () => {
               When you have completed all the sections in the lesson, you will
               be able to download the lesson resources, including the full
               lesson plan (which includes all sections), editable slides and
-              starter and exit quizzes. You can also share your lesson as a
-              link, which can be viewed online.
+              starter and exit quizzes, a worksheet with your practice tasks and
```

Comment:
I feel like this new paragraph is a bit clunky and has at least one too many 'and' in there, possibly this one:
editable slides and starter and exit quizzes, as we then use another one at the end of the sentence. May just need confirmation of how we want the actual section to read. Seems like it could be an intro sentence, list and then sharing outro sentence, but it also kind of replicates the list that is already up above. @hannah-bethclark